mass_mailing_custom_unsubscribe/models/mail_mass_mailing_list.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class MailMassMailing(models.Model):
+    _inherit = "mail.mass_mailing.list"
+
+    not_cross_unsubscriptable = fields.Boolean(
```

Comment:
I personally dislike the negative booleans, where a "no" means "yes". They lead to double negation blocks such as:

```python
if not record.not_cross_unsubscriptable:
    # It's cross_unsubscriptable! 😵
```

All you'd need is call this field `cross_unsubscriptable`, do code changes, add `default=True` and add a migration script that inverts its value wherever present.

Well, this is not a strict requirement, since this was there before as it is now, but I'd really appreciate it.

Reply:
@Yajo, although I somehow agree on the basis, we can't spend more time in this migration, so we can put this in the ROADMAP and change it in other round

Reply:
I think @pedrobaeza spend time in this migration, it will save us a lot of time after the migration.

@victormmtorres I wish after our meeting | training of today can review this part meticulously

Reply:
What do you mean? This is only semantic, so the change is optional. The feature is there.